csharp/ql/test/query-tests/API Abuse/FormatInvalid/FormatMissingArgument.cs

```diff
@@ -28,5 +29,59 @@ void helper(string format)
         String.Format(format, 0); // $ Alert Sink
     }
 
+    void TestCompositeFormatMissingArgument()
+    {
+        var format0 = CompositeFormat.Parse("{0}");
+        var format1 = CompositeFormat.Parse("{1}"); // $ Source
```

Comment:
Again, it is better to uniquely identify the sources so we can match them up against the alerts.